core/store/src/cold_storage.rs

```diff
@@ -68,6 +70,10 @@ pub fn update_cold_db<D: Database>(
         }
     }
 
+    let end_time = Clock::instant();
+    let duration = end_time.saturating_duration_since(start_time);
+    metrics::COLD_COPY_DURATION.observe(duration.as_secs_f64());
```

Comment:
`Histogram` has a special function for this. Do this at the beginning of the function instead:

```
let _timer = metrics::COLD_COPY_DURATION.start_timer();
```